common/skip links/reports/pre-ally-1.json

```
@@ -0,0 +1,589 @@
+{
+    "@context": "https://wet-boew.github.io/vocab/context/2023/assessment-report-en.json-ld",
+
+    "@type": ["earl:Assertion", "acr:AssessmentReport"],
+
+    "earl:subject": {
+        "@id": "_:subject",
+        "dct:references": "https://wet-boew.github.io/GCWeb/sites/skiplinks/skiplinks-en.html",
+        "@type": [
+            "earl:TestSubject",
+            "schema:WebPage"
+        ],
+        "dct:description": "Accessibility conformance reports of Skip links",
+        "earl:pointer": {
+            "@type": "oa:CssSelector",
+            "@value": "gc-features"
```

```suggestion
            "@value": ".wb-slc"
```

---

This CSS selector is better because the skip link do include the nav element

```suggestion
            "@value": "nav:has( #wb-tphp )"
```